src/states/elder/mod.rs

```diff
@@ -342,8 +342,9 @@ impl Elder {
                 debug!("{} Failed to send Merge: {:?}.", self, err);
             }
         }
-        if let Some(merged_info) = self.chain.try_merge()? {
-            self.vote_for_signed_event(merged_info)?;
+        if let Some(_merged_info) = self.chain.try_merge()? {
+            // Cannot vote for this new section info as we do not have a BlsPublicKey available.
+            // self.vote_for_section_info(merged_info)?;
```

Comment:
I thought the `merge` logic shall be removed?

Reply:
Other PR.